routemodel/solar_rad/solcast-api-test.py

```diff
@@ -0,0 +1,26 @@
+import json
+import requests
+
+api_key = 'na'
```

Comment:
I'd recommend creating the `config.py` file that way you don't have to worry about accidentally pushing it up; then you just import it as `from config import SOLCAST_API_KEY`